assets/css/sensei-course-theme/quiz-graded.scss

```diff
@@ -0,0 +1,65 @@
+.sensei-course-theme-quiz-graded {
+	&__container {
+		border: 1px solid #DCDCDE;
```

Comment:
I think these styles could be with the other notice styles in `assets/css/sensei-course-theme/notices.scss`. Also, similar to the other ones, we can add to the div the classes `sensei-lms-notice` and `sensei-course-theme__frame` that introduces the default styles to the notice, like the background, border, font-family, and so on that is common for all of them. :)

Also, it makes me notice that we didn't add the class `sensei-course-theme__frame` to all notice templates (it's only in the `lesson-quiz-notice.php`). Could you add it as part of this PR too?

Reply:
Sure. I didn't think about this part as a notice in the beginning. The design is a bit different from other notices.

Updated. Now all notice templates use the same class naming convention. 69b48922c7a8c961d7a3562ac0a8118a2439ee51